lib/db/random.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+const randomBytes = require('../promise').promisify(require('crypto').randomBytes)
+
+//kept lowercase for backwards compatibility as the content-server normalizes them to lowercase
```

Comment:
@vbudhram it's done, could you pl. see if the comment is ok.

Reply:
1. Put a space between `// kept`
2. Remove the new line after it, so it is closer to the alphabet line
3. Something like: "The alphabet is lowercase for backwards compatibility with the previous keyspace and the content-server also normalizes the codes to lowercase on submit`?

Reply:
@deeptibaghel I think a comment more like vlad would be good. It is a little easier to read.